app/styles/custom-components/_vlc-agenda-items.scss

```diff
@@ -199,3 +192,52 @@ $i: 0;
   opacity: 1;
   color: $napier-green;
 }
+
+.vlc-agenda-items__body {
+  align-items: center;
+  padding: 1.2rem 1.8rem;
+
+  .vlc-agenda-items__body-padding {
+    padding-bottom: 1.5rem;
+  }
```

Comment:
You're using BEM to specify a child. We do that to prevent nesting. Nesting will add higher specificity scores than desired. To fix this, you can place this block of code alongside the other selectors instead of nesting it.